src/main/java/org/folio/circulation/domain/LoanRepresentation.java

```diff
@@ -66,8 +71,32 @@ private static void addAdditionalServicePointProperties(
     spSummary.put("discoveryDisplayName", servicePoint.getDiscoveryDisplayName());
     spSummary.put("description", servicePoint.getDescription());
     spSummary.put("shelvingLagTime", servicePoint.getShelvingLagTime());
-    spSummary.put("pickupLocation", servicePoint.getPickupLocation());    
-    
+    spSummary.put("pickupLocation", servicePoint.getPickupLocation());
+
     loanRepresentation.put(fieldName, spSummary);
   }
+
+  private void additionalBorrowerProperties(
+    JsonObject loanRepresentation,
+    User borrower,
+    String fieldName
```

Comment:
Given there aren't any scenarios where this would be different, why is the name of the property being used for the extended borrower information provided as a parameter?

Reply:
Only for consistency with the existing code.

Reply:
Ok. As it is unnecessary, please inline the parameter.